### ISSUE TITLE
Fails to install apkm 

### ISSUE THREAD

**Issue by assembleDebug** [MuntashirAkon/AppManager#927]
### Please check before submitting an issue

- [X] I know what my device, OS and App Manager versions are
- [X] I know how to take logs
- [X] I know how to reproduce the issue which may not be specific to my device

### Describe the bug

AM just closes the installation dialogue whenever we want to install apkm bundles. I do not have this problem with "apks" files. See attached logs, it clearly shows the problem.

### To Reproduce

1. Download any apkm bundle from apkmirror
2. Using any file manager, tap on apkm file and then tap on the AM's "Install" option.
3. The installer closes itself instantly.

### Expected behavior

Should show the AM installer.

### Logs

```
2022-11-10 19:46:47.989 D/PIA: On create, intent: Intent { act=android.intent.action.VIEW cat=[android.intent.category.BROWSABLE] dat=content://com.mixplorer.silver.file/... typ=application/apkm flg=0x8000c3 cmp=io.github.muntashirakon.AppManager.debug/io.github.muntashirakon.AppManager.apk.installer.PackageInstallerActivity (has extras) }
2022-11-10 19:46:48.031 E/ApkFile: Invalid mime: application/apkm
2022-11-10 19:46:48.099 E/PIVM: Couldn't fetch package info
io.github.muntashirakon.AppManager.apk.ApkFile$ApkFileException: org.json.JSONException: No value for info_version
	at io.github.muntashirakon.AppManager.apk.ApkFile.<init>(ApkFile.java:345)
	at io.github.muntashirakon.AppManager.apk.ApkFile.createInstance(ApkFile.java:134)
	at io.github.muntashirakon.AppManager.apk.installer.PackageInstallerViewModel.lambda$getPackageInfo$0$io-github-muntashirakon-AppManager-apk-installer-PackageInstallerViewModel(PackageInstallerViewModel.java:75)
	at io.github.muntashirakon.AppManager.apk.installer.PackageInstallerViewModel$$ExternalSyntheticLambda1.run(Unknown Source:6)
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:463)
	at java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1137)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:637)
	at java.lang.Thread.run(Thread.java:1012)
Caused by: org.json.JSONException: No value for info_version
	at org.json.JSONObject.get(JSONObject.java:398)
	at org.json.JSONObject.getLong(JSONObject.java:523)
	at io.github.muntashirakon.AppManager.apk.splitapk.ApksMetadata.readMetadata(ApksMetadata.java:111)
	at io.github.muntashirakon.AppManager.apk.ApkFile.<init>(ApkFile.java:342)
	... 8 more
```

### Device info

- Device: Note 10 pro
- OS Version: Android T
- App Manager Version: Latest from debug channel
- Mode: Root

### Additional context

None

**Comment by MuntashirAkon** [comment]
Yikes! Do they also use `info.json` in APKM?

**Comment by MuntashirAkon** [comment]
Fixed in 42df9351e93ddfffe4f729ed58b705e235c87ac8
